src/PhpSpreadsheet/Worksheet/BaseDrawing.php

```diff
@@ -508,4 +516,24 @@ public function __clone()
             }
         }
     }
+
+    /**
+     * @param Hyperlink $pHyperlink
+     */
+    public function setHyperlink(Hyperlink $pHyperlink = null)
+    {
+        $this->hyperlink = $pHyperlink;
+    }
+
+    /**
+     * @return Hyperlink
+     */
+    public function getHyperlink()
+    {
+        if ($this->hyperlink === null) {
+            $this->hyperlink = new Hyperlink();
```

Comment:
Why do you create new Hyperlink if it does not exist, instead of returning `null` ?